src/tox/tox_env/runner.py

```diff
@@ -121,9 +121,10 @@ def _call_pkg_envs(self, method_name: str, *args: Any) -> None:
             with package_env.display_context(suspend=self._has_display_suspended):
                 getattr(package_env, method_name)(*args)
 
-    def _clean(self, force: bool = False) -> None:
-        super()._clean(force)
-        self._call_pkg_envs("_clean")  # do not pass force along, allow package env to ignore if requested
+    def _clean(self, transitive: bool = False) -> None:
+        super()._clean(transitive)
```

Comment:
The parent class's `_clean` does not evaluate the `transitive` argument, but maybe it is better to pass it anyway, as it is probably not a good idea to have different method signatures between parent and child class.

Reply:
You can't have different interface and expect the call to work so we have no option on this. Furthermore just because it's not using today doesn't mean that will never use it.